### PR TITLE
Html zip

### PR DESCRIPTION
**Pull request by jburel**
This PR re-introduces the download of the documentation
This time as zip not a pdf.

### CODE REVIEW

**Comment by hflynn** [comment]
line 80 column 72 - Warning: missing `</a> before <a>`

File: omero_downloads.html [hunk 1]
```diff
@@ -75,8 +75,9 @@
                     </li>
 
                     <li>
-                        <p>Full documentation is available as <a
-                                href="@DOC_URL@">web documentation</a> and there are
+                        <p>Full documentation is available to <a
+                                href="@DOC_URL@">read on the web</a> or to 
+                                <a href="@DOC"/>download as zipped html</a> and there are
```
Comment (hflynn) [inline omero_downloads.html]: Think you'll need to remove the `/` in the `<a href="@DOC">` tag too

**Comment by hflynn** [comment]
👍 provided it's green this time

**Comment by jburel** [comment]
It is green now 😉

**Comment by hflynn** [comment]
@jburel do you want to edit this once my renaming PRs are merged or shall i merge and open a new PR?

**Comment by jburel** [comment]
@hflynn name updated

File: gen.py [hunk 2]
```diff
@@ -55,7 +55,8 @@ def usage():
         ("SERVER36", "artifacts/OMERO.server-@VERSION@-ice36-@BUILD@.zip"),
         ("PYTHON36", "artifacts/OMERO.py-@VERSION@-ice36-@BUILD@.zip"),
         ("JAVA36", "artifacts/OMERO.java-@VERSION@-ice36-@BUILD@.zip"),
-        ("DOCS", "artifacts/OMERO.docs-@VERSION@-ice36-@BUILD@.zip"),
+        ("DOCS", "artifacts/OMERO.apidoc-@VERSION@-ice36-@BUILD@.zip"),
+        ("DOC", "artifacts/OMERO-doc-@VERSION@.zip"),
```
Comment (hflynn) [inline gen.py]: Hmm, I hadn't noticed this was inconsistent, shall i change the docs PRs to make it OMERO.doc-@version@?

Comment (jburel) [reply gen.py]: It will diverge from the B-F one but will be better for omero cc @sbesson

Comment (hflynn) [reply gen.py]: BF has a different pattern of dashes and underscores, I think keeping OMERO all consistently OMERO.component-version is more desirable. I'll push that change to https://github.com/openmicroscopy/ome-documentation/pull/1604

Comment (jburel) [reply gen.py]: happy with that

**Review by hflynn — APPROVED** [review]
Docs PR updated to match, this can get final review during the milestone release but changes look good

**Pull request merged**